docs/dag-serialization.rst

```diff
@@ -21,7 +21,7 @@
 DAG Serialization
 =================
 
-In order to make Airflow Webserver stateless (almost!), Airflow >=1.10.7 supports
+In order to make Airflow Webserver stateless, Airflow >=1.10.7 supports
```

Comment:
Somewhere we should mention that 1.10.10 is needed for this last bit.

Reply:
Updated